### PR TITLE
Refactor naming of TwoLibsOneProjectInteraction_DiscreteLibs_SingleProject

### PR DESCRIPTION
**Pull request by cormensratio**
Removes underscores from repository name and adapts project name to conform with snake style naming.

### CODE REVIEW

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- varats/varats/projects/cpp_projects/TwoLibsOneProjectInteractionDiscreteLibsSingleProject.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6248280&bid=20348258)

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6248280&bid=20348258)

File: varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-"""Project file for TwoLibsOneProjectInteraction_DiscreteLibs_SingleProject."""
+"""Project file for
+two_libs_one_project_interaction_discrete_libs_single_project."""
```
Comment (vulder) [inline varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py]: Shouldn't this also be spelled in camel case?

**Review by vulder — CHANGES_REQUESTED** [review]

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6248280&bid=20348258)

File: varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py (hunk 1 above)
Comment (cormensratio) [reply varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py]: Pylint complains that it should be snake case style.

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6248280&bid=20348258)

**Review by vulder — APPROVED** [review]

File: varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py (hunk 1 above)
Comment (vulder) [reply varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py]: Wut? And if you write something like `Project implementation for the CamleCase test library`

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6248280&bid=20348258)

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6248280&bid=20348258)

File: varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py (hunk 1 above)
Comment (cormensratio) [reply varats/varats/projects/cpp_projects/two_libs_one_project_interaction_discrete_libs_single_project.py]: Maybe pylint is triggered by the last word of the doc string. Wrapped in a sentence, pylint does not complain anymore.

**Pull request merged**